Readme.md

```diff
@@ -4,61 +4,14 @@ The next generation OONI Probe Command Line Interface.
 
 ## Development setup
 
-Be sure you have golang >= 1.8.
+Be sure you have golang >= 1.12. We use Go modules. Make sure you
+don't have GOPATH set.
```

Comment:
Ah I understand now that you have upgraded to use go modules. I am not too familiar with this workflow.

Reply:
Yes, indeed. Well, the upgrade path is very smooth. Unset `GOPATH`. Use `go build`.

Reply:
Yeah, the only thing I need to figure out is how to make this play well with the several other projects I have which may not work with such recent versions of golang (I am still on 1.11).

Reply:
I can help in doing the update! :-)